notify/drivers/sfdc.py

```diff
@@ -0,0 +1,275 @@
+# Copyright 2016: Mirantis Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import json
+import logging
+from xml.dom import minidom
+
+import requests
+from requests.packages.urllib3 import exceptions as urllib_exc
+
+from notify import config
+from notify import driver
+
+requests.packages.urllib3.disable_warnings(urllib_exc.InsecureRequestWarning)
+
+
+LOG = logging.getLogger("sfdc")
```

Comment:
You should not name modules like that because it breaks hierarchical structure of logging and makes it painful to configure it later. So, I would suggest to use `__name__` here and this module will use the `notify.drivers.sfdc` logger. This convention will provide an the easiest way to configure logger, e.g. to set level, formatters or streams  for all drivers `notify.drivers.*` or just for this specific driver `notify.drivers.sfdc` in the future.